modules/manage/pages/security/encryption.adoc

```diff
@@ -326,6 +326,24 @@ Schema Registry and HTTP Proxy connect to Redpanda over the Kafka API. If you co
 * xref:./authentication.adoc#configure-schema-registry-and-http-proxy-to-connect-to-redpanda-with-sasl[Configure Schema Registry and HTTP Proxy to connect to Redpanda with SASL]
 * xref:./listener-configuration.adoc[Configure Listeners]
 
+=== Manage the minimum TLS version
+
+Redpanda sets the minimum TLS version for all clusters to 1.2, using the xref:reference:properties/cluster-properties.adoc#tls_min_version[`tls_min_version`] cluster configuration property. This prevents client applications from negotiating a downgrade to the TLS version when they make a connection to a Redpanda cluster.
+
+You can update the minimum TLS version of your clusters to `v1.0`, `v1.1` or `v1.3` using `rpk`. Replace the placeholder in brackets. 
+
+[,bash]
+----
+rpk cluster config set tls-min_version=<version-number>
```

Comment:
```suggestion
rpk cluster config set tls_min_version=<version-number>
```

version number can be:
* `v1.0`
* `v1.1`
* `v1.2`
* `v1.3`

Reply:
@michael-redpanda. The command is now updated and all versions are listed on line 333. Is that OK?